public/app/features/dashboard/dashgrid/PanelStateWrapper.tsx

```diff
@@ -582,33 +584,68 @@ export class PanelStateWrapper extends PureComponent<Props, State> {
     });
 
     // for new panel header design
+    const containerClassNamesNewPanelChrome = classNames({
+      'panel-container--absolute': isSoloRoute(locationService.getLocation().pathname),
+      'panel-container--transparent': transparent,
+      'panel-container--no-title': this.hasOverlayHeader(),
+      [`panel-alert-state--${alertState}`]: alertState !== undefined,
+    });
+
```

Comment:
??  the point of the new PaneChrome is to replace all these

Reply:
I added them because I was not getting the right colors for the alert states(firing: red, pending: yellow, ok: green) and I did not want to implement that logic on this PR. I will check the code handling the alerts and see how much work would require.

Reply:
Alert state is a title item right?  Not sure why a class in an outer element affects the color of a title item

Reply:
yes it's a title item, as far as I see we are giving the colors using some global alert CSS https://github.com/grafana/grafana/blob/23956557d8c6a119b7de5be5c42024e29634d002/public/sass/pages/_alerting.scss#L47

Reply:
but add the correct class in https://github.com/grafana/grafana/pull/60514/files#diff-72fa0ae4d8ddfb483e3ccb61c7bbc3e17ed388182132ebe6a89446414a111daeR19 

or color it via emotion styles in PanelHeaderTitleItems.tsx 

Not by adding these legacy classes to wrapper div here, sort of defeats the purpose of PanelChrome :)

Reply:
@torkelo, I'll add the correct classes for alerts coloring in `PanelHeaderTitleItems` after I add them to `GrafanaTheme2` in order to follow the dark/light mode differences. 
This will take care of `[panel-alert-state--${alertState}]: alertState !== undefined`.

`'panel-container--no-title': this.hasOverlayHeader()` will be removed entirely, it is handled by @axelavargas's implementation of `timeshift` prop.

This way of styling the panel because of its different properties has to go indeed. However we are aiming to finish up the functionality of title items, so that we can implement the **hovering header** next.
So `'panel-container--absolute': isSoloRoute(locationService.getLocation().pathname)` and `'panel-container--transparent': transparent` will stay in this legacy form for now and will be fixed in a separate PR.

Reply:
@polibb why do we need 'panel-container--absolute': isSoloRoute(locationService.getLocation().pathname) and 'panel-container--transparent': ? can't we skip the section div for now and add transparent display mode and changes for solo route in follow-ups instead?

Reply:
> after I add them to GrafanaTheme2 in order to follow the dark/light mode differences.

What do you need to add to GrafanaTheme2? I don't see any need to add any new things to theme (it already has theme.colors.error theme.colors.success)

Reply:
>What do you need to add to GrafanaTheme2? I don't see any need to add any new things to theme (it already has theme.colors.error theme.colors.success)

Yes, sorry, I spoke before checking if we have all of them covered. We do 😊

Reply:
>can't we skip the section div for now and add transparent display mode to PanelChrome and changes for solo route in follow-ups instead?

Yes, pushed changes to remove any global styles and will leave transparent and solo panel functionalities for followups.